Testscripts/Windows/GPU-DRIVER-INSTALL.ps1

```diff
@@ -245,7 +258,14 @@ function Main {
         # The expected ratio is 1 GPU adapter for every 6 CPU cores
```

Comment:
Update this comment?

Reply:
done in 12986dc8187cb9bc52dde5b41a2b61feb65551b9